lab1/app.css

```diff
@@ -0,0 +1,101 @@
+body {
+    background-color: black;
+}
+
+div {
+    width: 1100px;
+    margin: auto;
+}
+
+.main-vertical-container {
+    display: flex;
+    height: 900px;
+    flex-direction: column;
+}
+
+.main-vertical-container>div {
+    margin: 10px 0;
+    /*UNCOMMENT TO SHOW BORDERS*/
+    /*border: solid white 3px;*/
```

Comment:
commented out code should not be in the final version of code. It defeats the purpose of version control.